ddl/db_test.go

```diff
@@ -620,6 +635,21 @@ LOOP:
 				s.mustExec(c, sql)
 			}
 			num += step
+		case <-ticker2.C:
+			if !startReorganization {
+				is = s.dom.InfoSchema()
+				tbl, err = is.TableByName(schemaName, tableName)
+				c.Assert(err, IsNil)
+				for _, idx := range tbl.Meta().Indices {
+					if idx.Name.L == "c3_index" && idx.State == model.StateWriteReorganization {
+						startReorganization = true
+					}
+				}
+			}
+			if startReorganization {
+				workerCnt := rand.Intn(8) + 8
+				s.mustExec(c, fmt.Sprintf("set @@tidb_ddl_reorg_worker_cnt=%d", workerCnt))
```

Comment:
How can we ensure the worker count is changed to `workerCnt`?

Reply:
em... I check the log when do manual test to know the worker count is changed... 😂